src/rpc/blockchain.cpp

```diff
@@ -1532,6 +1532,25 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
     return ret;
 }
 
+UniValue savemempool(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0) {
+        throw std::runtime_error(
+            "savemempool\n"
+            "\nDumps the mempool to disk.\n"
+            "\nExamples:\n"
+            + HelpExampleCli("savemempool", "")
+            + HelpExampleRpc("savemempool", "")
+        );
+    }
+
+    if (!DumpMempool()) {
+        throw JSONRPCError(RPC_MISC_ERROR, "Unable to dump mempool to disk");
+    }
+
+    return true;
```

Comment:
This return value is not documented. And I think we shouldn't return anything on this rpc.

Reply:
Right, since on failure an exception is raised, on success it can be void.

Reply:
Right, since on failure an exception is raised, on success it can be void.

Reply:
Yes, it appears that many RPCs that perform an action which is expected to succeed return `NullUniValue` on success. See several of the methods in src/rpc/net.cpp for example.